libs/langchain/langchain/document_loaders/youtube.py

```diff
@@ -177,7 +179,7 @@ def from_youtube_url(cls, youtube_url: str, **kwargs: Any) -> YoutubeLoader:
         video_id = cls.extract_video_id(youtube_url)
         return cls(video_id, **kwargs)
 
-    def load(self) -> List[Document]:
+    def load(self, duration: Optional[float] = None) -> List[Document]:
```

Comment:
this should not be passed into load

Reply:
Hey It is an optinal variable and when its not given its defaulted to None and that results in previous behaiviour, that was my intention, I'm not sure how i would be able to achieve that without the duration variable

Reply:
okay so ill just move the duration to init method and reference it from there got it thank you!